test/commons/text/is-human-interpretable.js

```diff
@@ -1,63 +1,71 @@
 describe('text.isHumanInterpretable', function () {
   it('returns 0 when given string is empty', function () {
-    var actual = axe.commons.text.isHumanInterpretable('');
+    const actual = axe.commons.text.isHumanInterpretable('');
     assert.equal(actual, 0);
   });
 
-  it('returns 0 when given string is a single character that is blacklisted as icon', function () {
-    var blacklistedIcons = ['x', 'i'];
-    blacklistedIcons.forEach(function (iconText) {
-      var actual = axe.commons.text.isHumanInterpretable(iconText);
+  it('returns 0 when given string is a single alpha character', function () {
+    const singleCharacterExamples = ['i', '×'];
```

Comment:
Should add back the `x` character instead of replacing it with the multiplication symbol.

```suggestion
    const singleCharacterExamples = ['i', 'x', '×'];
```